internal/pkg/cli/deploy/deploy.go

```diff
@@ -61,9 +62,19 @@ const (
 )
 
 var (
-	aliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
+	rdwsAliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
 		color.HighlightCode("http.alias"),
 		color.HighlightCode("copilot app init --domain example.com"))
+	ecsALBAliasUsedWithoutDomainFriendlyText = fmt.Sprintf(`To use %s, your application must be:
+* Associated with a domain: %s
+* Or, using imported certificates to your environment: %s
+`,
```

Comment:
nit: to avoid super long lines
```suggestion
	ecsALBAliasUsedWithoutDomainFriendlyText = fmt.Sprintf(`To use %s, your application must be:
* Associated with a domain:
     %s
* Or, using imported certificates to your environment: 
     %s
`,
```